OpenRA.Game/Graphics/SpriteRenderer.cs

```diff
@@ -82,16 +82,20 @@ int2 SetRenderStateForSprite(Sprite s)
 				for (; secondarySheetIndex < ns; secondarySheetIndex++)
 					if (sheets[secondarySheetIndex] == secondarySheet)
 						break;
+
+				// If neither sheet has been mapped both index values will be set to ns.
+				// This is fine if they both reference the same texture, but if they don't
+				// we must increment the secondary sheet index to the next free sampler.
+				if (secondarySheetIndex == sheetIndex && secondarySheet != sheet)
+					secondarySheetIndex += 1;
```

Comment:
`secondarySheetIndex++;` ?

Reply:
I considered that, but went with this to be consistent with the two other cases below.

I could change all three to ++, but i'm not sure its worth the effort?

Reply:
Hum, looks like we are using `+= 1` 25 times in the code base and `++` 195` times, imo ++ looks better so would be nice to replace all with this.